src/main/scala/com/lucidworks/spark/SolrRelation.scala

```diff
@@ -668,34 +672,26 @@ class SolrRelation(
 
     // build up a list of updates to send to the Solr Schema API
     val fieldsToAddToSolr = new ListBuffer[Update]()
+    val fieldsRequested = new ListBuffer[String]()
```

Comment:
note, this is because unpacking the Update object from fieldsToAddToSolr is a total pita once it's converted to an AddField object